tokio-reactor/README.md

```diff
@@ -2,6 +2,14 @@
 
 Event loop that drives Tokio I/O resources.
 
+This crate is **deprecated in tokio 0.2.x** and has been moved and refactored
```

Comment:
We should probably call out here that its be renamed from `reactor` to `driver`.